NuGet.config

```diff
@@ -3,7 +3,9 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <add key="darc-pub-dotnet-core-setup-b7dc9ed" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-b7dc9ed2/nuget/v3/index.json" />
     <add key="darc-pub-dotnet-core-setup-7d57652" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-7d57652f/nuget/v3/index.json" />
```

Comment:
@mmitche Should we be merging these new package sources from release/3.1 to master? What about the darc-pub-dotnet-core-setup-7d57652 source that's already there?

Reply:
@halter73 no, there's no reason for stabilization-specific branches in the 'master' branch.